wagtail/admin/templates/wagtailadmin/shared/status_tag.html

```diff
@@ -0,0 +1,11 @@
+{% if url %}
+    <a href="{{url}}" target="_blank" rel="noreferrer" class="status-tag {% if classname %}{{classname}}{% endif %}" {% if title %}title="{{title}}"{% endif %}>
+        {% if hidden_label %}
+            <span class="visuallyhidden">{{hidden_label}}</span> {% if label %}{{ label }}{% endif %}
+        {% else %}
+            {% if label %}{{ label }}{% endif %}
+        {% endif %}
+    </a>
+{% else %}
+    <span class="status-tag {% if classname %}{{classname}}{% endif %}">{% if hidden_label %}<span class="visuallyhidden">{{hidden_label}}</span> {% if label %}{{ label }}{% endif %} {% else %}{% if label %}{{label}}{% endif %}{% endif %}</span>
```

Comment:
this line seems overly convoluted, I think we can change it to

```suggestion
    <span class="{% classnames 'status-tag' classname %}">
        {% if hidden_label %}<span class="visuallyhidden">{{hidden_label}}</span>{% endif %}
        {% if label %}{{ label }}{% endif %}
    </span>
```